mieval_los_mu0: 2.56
mieval_los_mu: -0.06
mieval_los_sigma2: 0.1479

mieval_nlos_mu0: -0.2596
mieval_nlos_mu: -0.4533
mieval_nlos_sigma2: 1.33
